setup/ubuntu16.04_docker_alpine3.4/resources/Dockerfile.j2

```diff
@@ -0,0 +1,45 @@
+FROM alpine:3.3
+
+ENV LC_ALL C
+ENV USER {{ server_user }}
+ENV JOBS {{ server_jobs | default(ansible_processor_vcpus) }}
+ENV HOME /home/{{ server_user }}
+ENV NODE_TEST_DIR /home/{{ server_user }}/tmp
+ENV PATH /usr/lib/ccache/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+ENV NODE_COMMON_PIPE /home/{{ server_user }}/test.pipe
+ENV OSTYPE linux-gnu
+ENV DESTCPU x64
+ENV ARCH x64
+
+RUN apk add --no-cache \
+        libstdc++ \
+    && apk add --no-cache --virtual .build-deps \
+        binutils-gold \
+        curl \
+        g++ \
+        gcc \
+        gnupg \
+        libgcc \
+        linux-headers \
+        make \
+        paxctl \
+        python \
+        tar \
+        ccache \
+        openjdk8 \
+        git
+
+RUN addgroup -g 1000 {{ server_user }}
+
+RUN adduser -G {{ server_user }} -D -u 1000 {{ server_user }}
+
+VOLUME [ "/home/{{ server_user }}/" ]
+
+CMD su {{ server_user }} -c '\
```

Comment:
THANKYOU! I didn't know this was a Dockerfile thing, must be new I suppose. This has solved the 3.4 problem (plus I'll be changing a bunch of other Dockerfiles now I know this exists).